vtm-web-app/war/index.html

```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
     <title>vtm-gdx</title>
+    <link rel="icon" type="image/svg+xml" href="assets/launcher/ic_vtm.svg" sizes="any">
```

Comment:
That svg is missing?

Reply:
Yes, `/vtm-web-app/assets/launcher` was in .gitignore. I added it now to the vtm assets, cause I think it is logo for all modules (?). `vtm-themes/resources/assets` or `/vtm-web-app/war` maybe possible, too...

Reply:
`vtm-web-app/assets` is constructed from all modules during web app build.
I moved the svg in `vtm-web-app/resources/assets/logo/` where web app assets exist.